portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,29 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Laura!");
-  }
+    private List<String> messages;
+
+    @Override
+    public void init() {
+        messages = new ArrayList<String>();
+        messages.add("How are you?");
+        messages.add("How is the weather today?");
+        messages.add("What are you doing?");
+    }
+
+    @Override 
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        // Converting Array List to JSON
+        String json_version = convertToJson(messages);
```

Comment:
Use lowerCamelCase for variable names (https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names)